ons_alpha/jinja2/templates/pages/bulletins/bulletin_page.html

```diff
@@ -100,6 +98,5 @@ <h3 class="ons-u-mb-no ons-u-fs-r--b">Phone</h3>
     <script async src="{{ static('js/third-party/pym.min.js') }}"></script>
     <script async src="{{ static('js/blocks/embeddable.js') }}"></script>
 
-    {# HACK: We can't use the built-in inclusion tag, so import the template _directly_. #}
```

Comment:
Question: Why have these comments been removed?

Reply:
Apologies this was an accident, I have put this back in my recent commit.